@here/harp-mapview/lib/text/TextStyleCache.ts

```diff
@@ -441,19 +461,48 @@ export class TextStyleCache {
                     style.wrappingMode === "Word"
                         ? WrappingMode[style.wrappingMode]
                         : WrappingMode.Word,
-                verticalAlignment:
-                    style.vAlignment === "Above" ||
-                    style.vAlignment === "Center" ||
-                    style.vAlignment === "Below"
-                        ? VerticalAlignment[style.vAlignment]
-                        : VerticalAlignment.Center,
-                horizontalAlignment:
-                    style.hAlignment === "Left" ||
-                    style.hAlignment === "Center" ||
-                    style.hAlignment === "Right"
-                        ? HorizontalAlignment[style.hAlignment]
-                        : HorizontalAlignment.Center
+                verticalAlignment,
+                horizontalAlignment
             }
         };
     }
+
+    private resolvePlacementAndAlignment(
+        hAlignment?: string,
+        vAlignment?: string,
+        hPlacement?: string,
+        vPlacement?: string
+    ): {
+        horizontalAlignment: HorizontalAlignment;
+        verticalAlignment: VerticalAlignment;
+    } {
+        // Parse placement properties if available at they have higher priority in case of clash.
+        const horizontalPlacement =
+            hPlacement === "Left" || hPlacement === "Center" || hPlacement === "Right"
+                ? HorizontalPlacement[hPlacement]
+                : undefined;
+        const verticalPlacement =
+            vPlacement === "Above" || vPlacement === "Center" || vPlacement === "Below"
```

Comment:
"Above" to "Top" and "Below" to "Bottom"

Reply:
Correct, sorry messed up during merge.